### DISCUSSION TITLE
Welcome to EjercicioGIT_viernes_11 Discussions!

### DISCUSSION THREAD

**Discussion by florezal754** [florezal754/EjercicioGIT_viernes_11#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by florezal754** [comment]
Hola, esto es una prueba

**Reply by hector117best** [reply]
vale creo que puedo tambine por aqui

**Reply by florezal754** [reply]
genial

**Reply by hector117best** [reply]
para subir el trabajo individual donde se guarda

**Reply by arsenelupincrypto** [reply]
dios mio no lo veia que lio
